test/models/tf_ov_model_zoo/run_infer_multiple.sh

```diff
@@ -24,11 +24,18 @@ while getopts “:m:b:d:h” opt; do
 done
 shift $((OPTIND-1))
 
+#==============================================================================
+#==============================================================================
+
 [ -f "$MANIFEST" ] || ( echo "Manifest not found: $MANIFEST !"; exit 1 )
 MANIFEST="$(cd "$(dirname "$MANIFEST")"; pwd)/$(basename "$MANIFEST")" # absolute path
 
 cd ${WORKDIR} || ( echo "Not found: $WORKDIR !"; exit 1 )
 echo "Dir: ${WORKDIR}"
+CSVFILE=benchmark_avg_infer_msec.csv
+rm $CSVFILE
```

Comment:
This will error out if the file doesn't exist.

Reply:
reverted